### PR TITLE
Display column names instead of indices for remainder columns in the transformers_ attribute of the SuperVectorizer

### PR DESCRIPTION
**Pull request by LeoGrin**
Solves #232 
Following discussion with @GaelVaroquaux in previous PR (#236), I've made the modification in the fit method of the SuperVectorizer.

### CODE REVIEW

File: dirty_cat/super_vectorizer.py [hunk 1]
```diff
@@ -427,7 +427,16 @@ def fit_transform(self, X, y=None):
         if self.verbose:
             print(f'[SuperVectorizer] Assigned transformers: {self.transformers}')
 
-        return super().fit_transform(X, y)
+        res = super().fit_transform(X, y)
+
+        # for the "remainder" columns, the ColumnTransformer transformers_ attribute
+        # contains the index instead of the column name so we convert it to the column name.
+        for i, tup in enumerate(self.transformers_):
+            name, enc, cols = tup  # Unpack
```
Comment (GaelVaroquaux) [inline dirty_cat/super_vectorizer.py:435]: Maybe let's do this only if len(cols) < 20, if you agree?

**Review by GaelVaroquaux — CHANGES_REQUESTED** [review]
Thank you. That's a great feature. I made a minor suggestion.

File: dirty_cat/super_vectorizer.py (hunk 1 above)
Comment (LeoGrin) [reply dirty_cat/super_vectorizer.py:436]: Oh yes, agreed!

**Review by GaelVaroquaux — APPROVED** [review]

**Comment by GaelVaroquaux** [comment]
Merged! Thanks

**Pull request merged**